### ISSUE TITLE
Integrate PNO into IAO

### ISSUE THREAD

**Issue by mbrochhausen** [information-artifact-ontology/IAO#237]
The Proper Name Ontology (http://purl.obolibrary.org/obo/iao/pno.owl) was created a few years back and was kept separate for multiple reasons, which at this point have all become obsolete. I propose to integrate PNO into IAO. I will happily join to IAO developer meetings to help coordinate that and I'll participate in the future to help further development of the identifier branch.

**Comment by hoganwr** [comment]
+1

**Comment by jamesaoverton** [comment]
This seems like the right approach to me.

**Comment by zhengj2007** [comment]
related issue: https://github.com/information-artifact-ontology/IAO/issues/236

**Comment by alanruttenberg** [comment]
I'm open to this, though I think it needs review and discussion. e.g.
"An identifier is an information content entity that is the outcome of a dubbing process and is used to refer to one instance of entity shared by a group of people to refer to that individual entity."

- family name is asserted to be identifier. But it's not clear that family name alone designates one instance. Makes more sense to me to have first and last name be /parts/ of a personal name.
- First name and given name vary across cultures. As I understand it, in China the "first name" is actually the second. This suggests to me that these should have a more specific label.
- Some of the terms seem to belong elsewhere, e.g. mechanical wave quality. Also, I don't understand the definition of that term.

I'm open to meeting some time to work through this.

**Comment by mbrochhausen** [comment]
Hi,

I agree with point one and two Alan raises. I did think I addressed the
aspect Alan raised in 2), but it appears I haven't. However, full agreement
with Alan that his needs to be addressed.
I am looking forward to his suggestion of where to put mechanical wave
quality. Needless to say I am absolutely open to discuss the definition.

I'd strongly prefer to hash these things out by e-mail or issue tracker as
my time is extremely limited and I would like to see this move forward.

Best,
Mathias

Best,
Mathias

**Comment by mbrochhausen** [comment]
Alan,

can you please specify the issue with 'given name'. I consciously avoided
the terms 'first name' and 'last name'. THe only reference to first name I
find related to 'given name' is in the definition:
"A given name, in Western contexts often referred to as a first name, is a
personal name that specifies and differentiates between members of a group
of individuals, especially in a family, all of whose members usually share
the same family name (surname). A given name is purposefully given, usually
by a child's parents at or near birth, in contrast to an inherited one such
as a family name"

I propose to change it to:
"A given name is a personal name that specifies and differentiates between
members of a group of individuals, especially in a family, all of whose
members usually share the same family name. A given name is purposefully
given, usually by a child's parents at or near birth, in contrast to an
inherited one such as a family name."
 In the same vein, I propose to change the definition of family name as
follows:

"A family name is typically a part of a person's name which has been
passed, according to law or custom, from one or both parents to their
children.'

**Comment by mbrochhausen** [comment]
Wait, hold on. I propose more regarding 'family name':
I propose to move to be a subclass of identifier and change the definition to
"An identifier that refers to a group a people associated with each other through ancestral or family relations."

**Comment by mbrochhausen** [comment]
Nowadays, I am quite sure that there is no necessary relationship between a family name and a person name or vice versa.
A good example of this is the Brazilian soccer player Pele. Pele is a personal name IMO, but it doesn't have a connection to a family name. Also, since family names identify groups that can be used and exist without actually being part of a person name.

**Comment by alanruttenberg** [comment]
Will respond to you points a little later. Regarding mechanical wave quality: In BFO2020 GDCs can be concretized in processes, something I had pushed for for a long time. I think in this context we could get rid of the physiology / physics aspects and define utterance process as a concretization  of some class of GDCs.  With that, we don't need (and I'd argue don't want) the term utterance. The are problematic in that it's not clear all utterances processes convey GDCs. Consider "ouch". 

There's been a need for some time to have 'communication process', which might be able to define relatively cleanly.   

It's also not clear to me why we have the anatomy term, which don't seem to used. We need Homo Sapiens, for personal name, though there's the question of whether machines can have names. 

For 'code set', the one-to-one relation to entities doesn't cover the case where an identifier denotes a class or universal. At least in the OWL version. There was a last minute change to the BFO-2020 related to this where Barry had used the term entity in the spec to mean particular or universal. But usage until now, I believe, and certainly in prior usage in OWL has meant particulars.'

Even so, relations are sometimes collected into a code set, and relations are not considered entities.

A specific proposal - starting point for discussion:
- Deprecate portion of energy and subs
- Deprecate oscillating
- Don't import material anatomical entity and subs
- Drop utterance but keep the other siblings
- Add communication process subtypes: spoken communication process, possibly 'composing of written communication process'. Communication processes have a author and target (both roles). spoken communication processes have parts that are concretizations of graphemes, though this rules out languages that are only spoken. 
- Deprecate utterance. It's arguably out of scope in that includes non language speech. It's also a bit confusing in that utterance process *is* focused on a language.
- Deprecate utterance process in favor of communication process (possibly 'spoken' depending on your intent)  
Add examples of usage for remaining terms.

Potential issues: 
- Speaking to yourself, writing a journal. Arguably, in these cases the same thing has the author/target roles.  
- Languages that are only spoken

Note: I haven't reviewed the relations yet.

**Comment by mbrochhausen** [comment]
Ok, it seems the scope of criticism/required changes is still a
moving target. I'll wait until your review is completed and then address
your comments.

Thanks, this is helpful.

Best,
Mathias

**Comment by mbrochhausen** [comment]
PS: I like the idea of getting utterance away from the physio-anatomical
aspect!
I might be wrong, but I think we recently did define 'communication
process" in a language module for OMRSE. Bill (Hogan)?

**Comment by alanruttenberg** [comment]
Another thought: Name is proper name ontology but there's no proper name class. For instance the current ontology would not cover a place name.

**Comment by hoganwr** [comment]
Yes, we defined 'communication process' in OMRSE in consultation with MFO.

**Comment by alanruttenberg** [comment]
Def: A process in which some participant shares some information content entity about some state of that participant with some other participant.

I'm not sure that the about part is broad enough. If I send a chain letter to someone it isn't about  "some state of that participant with some other participant."
Similarly reading a bed time story to your kid.

**Comment by mbrochhausen** [comment]
1. PNO is called the Proper Name Ontology for historical reasons. The class
that is labeled 'identifier' has 'proper name' as an alternative term now.
Originally, the class was labeled 'proper name', which was deemed to be the
central class of the ontology, which took on the domain of names.
2. The ontology implements a theory of names put forward here Devitt and
Sterelny (
https://www.amazon.com/Language-Reality-2nd-Introduction-Philosophy/dp/0262540991/ref=sr_1_12?dchild=1&keywords=sterelny&qid=1608153647&sr=8-12).
I thought that was referenced in the ontology metadata, but it isn't. That
needs to be fixed. The definition of 'identifier' is still in accordance
with that book.
3. The change from 'proper name' to 'identifier' was done in communication
with IAO and OBO FOundry colleagues, so I am a tiny bit surprised that
seems to be a problem now.
4. I don't understand why place names supposedly do not fall under
identifiers. The definition reads: "An identifier is an information content
entity that is the outcome of a dubbing process and is used to refer to one
instance of entity shared by a group of people to refer to that individual
entity." (again, which is Devitt and Sterelny's definition of proper name).

**Comment by mbrochhausen** [comment]
1. I would like to keep the discussion of the OMRSE term for communication
out of this discussion, for the obvious reason that this is an OMRSE term
and we are discussing merging IAO and PNO.
2. I do not think communication in the broad sense belongs into IAO.
Communication ought to be part of a higher level social ontology.
3. I agree with Alan that the term communication can be used in natural
language to refer to something broader, but I will (despite the fact that
this discussion doesn't belong here) say that I think the OMRSE class is
valid and represents one fairly frequent usage of the term 'communication".
My proposal would be that whoever needs the broader sense provides a
definition and then checks whether there is no label that fits their class
except "communication". If that is the case OMRSE and the person with the
broader class need to coordinate who gets to use this label.

**Comment by zhengj2007** [comment]
Chris S., John J, Mark M, and I looked through the PNO terms. Generally we agreed with Alan’s comments. We think following terms are in IAO scope and may need to integrate into IAO.
IAO:0020000		identifier
IAO:0020001		grapheme
IAO:0020002		utterance
IAO:0020015		personal name
IAO:0020016		given name
IAO:0020017		family name
IAO:0020020		code set
IAO:0020010		dubbing process

However, term ‘code set’ and ‘dubbing process’ need to be improved. 

It needs to clarify distinction of ‘code set' with 'data set' and what is meant by code in label 'code set’. 
Here is the code set example and definition we found online for you to consider:
Under HIPAA, a "code set" is any set of codes used for encoding data elements, such as tables of terms, medical concepts, medical diagnosis codes, or medical procedure codes.
https://www.cms.gov/Regulations-and-Guidance/Administrative-Simplification/Code-Sets

Regarding ‘dubbing process’ , we think dubbing isn't an intuitive label based on the logical axiom. How about 'identifier creating process' or something like that.”?

[List of terms in PNO](https://docs.google.com/spreadsheets/d/14Ry7CI0CLcjy_qMRgARKrQaQdVkopSo3nGOsDv7z5hM/edit#gid=1390486785)

**Comment by mbrochhausen** [comment]
Hi,

thanks for taking the time to assess this. I do have a number of questions:


1. Could you please give a bit of rationale why in your minds writing
quality and writing bearer are out of scope for IAO.

2,
IAO:0020013
IAO:0020014

Why are those two object properties regarded out of scope?

3. Given the classes that are out of scope how do you propose we ensure
rigor and consistency for the existing classes?

Comments:

1. I have no objections to using "identifier creating process" as label for
the class. However, "dubbing process" (which is the linguistic term) should
be kept as an alternative term. I would also suggest to add "naming" as an
alternative term.

2. Not attempting to write a definition, but trying to clarify the
difference between code set and data set.

Here is an example of a code set:
"Raiders of the Lost Ark" - IJ_01
"Indiana Jones and the Temple of Doom"  - IJ_02
"Indiana Jones and the Last Crusade" - IJ_03
"Indiana Jones and the Crystal Skull" - IJ_04

According to IAO a data set is:
"A data item that is an aggregate of other data items of the same type that
have something in common. Averages and distributions can be determined for
data sets."

A data item is:
"An information content entity that is intended to be a truthful statement
about something (modulo, e.g., measurement precision or other systematic
errors) and is constructed/acquired by a method which reliably tends to
produce (approximately) truthful statements."

The list above is, in my mind, not a data item, since it is not something
that is empirically verifiable. Someone (in case me) decided to code the 4
Indiana Jones movies that way. There may be other ways to code them. I
think saying that any of the coding is "true" is committing a category
mistake.

Happy to work on the definition to make that clearer.

Best,
Mathias

**Comment by zhengj2007** [comment]
I checked 3 object properties defined in PNO:
IAO:0020013	is_borrowed_reference_to
IAO:0020014	is_fixing_reference_to
IAO:0020018	is designated by

They are not used in any ontology now. The inversion property of  IAO:0020018 'is designated by', IAO:0020012	obsolete_designates, is the only property that used in OBIB. However, it is deprecated in PNO.

So, I propose to deprecate PNO object properties that are not used unless we have clear use cases to make the PNO integration easily.

**Comment by hoganwr** [comment]
+1 to Mathias' comments and suggestions.

Alternatively, editor preferred term could be "identifier creating process"
and keep label as is.

Bill

**Comment by zhengj2007** [comment]
Discussed on March 22nd, 2021 OBI call.

We agreed to integrate following PNO terms to IAO:
IAO:0020000	identifier
IAO:0020001	grapheme
IAO:0020002	utterance
IAO:0020010	dubbing process
IAO:0020015	personal name
IAO:0020016	given name
IAO:0020017	family name
IAO:0020020	code set

We discussed PNO:identifier, PNO:code set, IAO:symbol, and think
- PNO:identifier is a subClassOf IAO:symbol
- PNO:code set is not a data item, it is a sibling of IAO:symbol

Current IAO_0000579 'centrally registered identifier registry' is a 'data set' which seems incorrect. It should be 'code set' (consider renaming as 'code map').

Consider relabel IAO:0020010 as 'identifier creating process' with alternative name 'dubbing process'. 

AI for Mathias: will create a branch, add PNO terms to IAO, make changes of IAO term parent when need, and make the pull request. 

We will set another OBI call to review the changes.

@mbrochhausen Could you please check the notes and correct me if anything wrong? Thanks!

**Comment by mbrochhausen** [comment]
That is what I remember, too.

I will also add "naming" as an alternative term for "identifier creating process".

**Comment by mbrochhausen** [comment]
Sorry for taking so long on this. But now working on implementing the changes, I am reluctant to make identifier a subclass to symbol. The reason is that the definition of symbol implies that symbols are in some form graphic representation ("mark(s) or character(s) "). But not all identifiers are marks or characters or consist of them. Societies without a written language still have names (which are intended to be instances of identifiers).

**Comment by zhengj2007** [comment]
Discussed on OBI call 2021-08-09. We agreed with Mathias that 'identifier' should be subClass of 'information content entity'.

**Comment by pfabry** [comment]
Hi,

Do you have an ETA for the IAO release including personal name, family name and given name ?

Thanks,
Paul Fabry

**Comment by mbrochhausen** [comment]
Paul, thanks for asking. We are working on creating a proposal file that contains PNO. Once that is done the IAO community will decide.

If you need this quickly, there is no inconsistency between PNO and IAO and hence, nothing should prevent you from just importing both ontologies into an OWL file.

Best,
Mathias

**Comment by jmwhorton** [comment]
I have created a branch (integration/PNO) containing the listed changes for PNO integration:

Edited IAO:0020000 identifier, expanded class specification.
Added IAO:0020001 grapheme.
Added IAO:0020002 utterance.
Added IAO:0020010 identifier creating process.
Added IAO:0020015 personal name.
Added IAO:0020016 given name.
Added IAO:0020017 family name.
Added IAO:0020020 code set.
Moved IAO:0000579 (centrally registered identifier registry) under 'code  set'.
Added OBI:0100026 organism.
Added NCBITaxon:9606 Homo sapiens.

I would like to point out that in addition to the specifically requested PNO classes, I also added 'organism' and 'Homo sapiens' as dependencies for 'personal name.'

@mbrochhausen, please feel free if there's any important information I may have missed.

**Comment by zhengj2007** [comment]
Reviewed [pull request 256 ](https://github.com/information-artifact-ontology/IAO/pull/256) on 2022-07-18 OBI call.

Following changes need to be made:
- remove logic axiom, 'is about' only 'Homo sapiens',  associated with 'person name'. So, also need to revert changes on import
- update 'code set' definition, make it more general
- move [IAO_0000579](http://purl.obolibrary.org/obo/IAO_0000579) 'centrally registered identifier registry' under 'code set'

**Comment by jmwhorton** [comment]
Also from 2022-07-18 OBI call:

- Remove IAO:0020002 utterance

**Comment by yargan** [comment]
Dear all, 
I have a question regarding the modelling of institution names in IAO.

Here is the argument:
(Premise 1) Proper name == identifier; (Premise 2) An institution name is a proper name (Ergo) An institution name is an identifier. 

However, an institution name, e.g., the example of usage provided in the OWL file “University of Colorado Denver School of Medicine”, is an instance of [iao:institutional identification](http://purl.obolibrary.org/obo/IAO_0000303). On the other hand, (1) it is a proper name, (2) it points out a specific entity, or say labels a specific institution/organisation. In this sense, it does not differ from a given name. Then, why was “institutional identification” preferred instead of “institutional identifier”? This also opens the ground for the next issue: the use of the terms “identifier” and “identification”. 

The term “identification” occurs twice in IAO (and PNO), both of which are subclasses of [iao:textual entity](http://purl.obolibrary.org/obo/IAO_0000300). In general, the term “identification” seems to refer to the process of recognising something or proving the identity of something. In that sense, an instance of a textual entity, say the written name on this article I am reading, is about an author, which helps me to identify them. When I am asked “what is the name of the author?” I use this piece of information and find the author. Like, “what is the name of the paper? ” I use the document title and say it. I always need to refer to a textual entity for it. 

Another point is the confusion surrounding the term [iao:personal name](http://purl.obolibrary.org/obo/IAO_0020015), its child class [iao:given name](http://purl.obolibrary.org/obo/IAO_0020016), and its sibling class [iao:family name](http://purl.obolibrary.org/obo/IAO_0020017). The content of Wikipedia does not reach a conclusion, but in IAO, the decision is that an instance of a given name is also an instance of a personal name, and a family name is unrelated to a personal name. If we assert that a family name can be part of a personal name, then a given name as well. It is difficult to represent a person's full name. Person Name Vocabulary can be utilised, though. See [PNV visualisation](https://www.lodewijkpetram.nl/vocab/pnv/doc/img/PNV_visualization.png). 

The last point I want to raise (and also learn from your experiences) is the formal definitions of equivalent classes for [iao:identifier](http://purl.obolibrary.org/obo/IAO_0020000) and [iao:identifier creating process](http://purl.obolibrary.org/obo/IAO_0020010).
Defn(identifier): information content entity AND (denotes SOME entity) AND  (is specified output of SOME identifier creating process)
Defn(identifier creating process) completely executed planned process AND (has specified output SOME identifier)
An equivalence class of an entity is written with entities and properties which are simpler/general/more primitive than it. These terms have one another in their definitions. Are these legitimate formulations? 

Thanks for your time and concern in advance! 
Best,

dilek
